views/marts/product/fct_active_servers.view.lkml

```diff
@@ -127,4 +127,23 @@ view: fct_active_servers {
     view_label: " * Metrics: Active Servers"
     drill_fields: [server_id]
   }
+
```

Comment:
Another approach would be to create a `dim_date` that contains all the information about the date. This hasn't been done so far as there was no need.

Should we go for this direction or keep the "quick and dirty" solution in this PR (needs to be tested)?

Reply:
or maybe keep just task 1 in the PR and punt task 2 for the `dim_date`, more long-term solution?

Reply:
Let's keep current PR as is, and when dim_date is ready make all required changes. The risk is that if these dimensions are removed, any dashboards/looks using them might break.